src/inabox/inabox-viewport.js

```diff
@@ -149,35 +151,91 @@ export class ViewportBindingInabox {
     /** @private @const {boolean} */
     this.useLayers_ = isExperimentOn(this.win, 'layers');
 
+    /** @private {?../../ads/inabox/position-observer.PositionObserver} */
+    this.topWindowPositionObserver_ = null;
+
     dev().fine(TAG, 'initialized inabox viewport');
   }
 
   /** @override */
   connect() {
-    this.listenForPosition_();
+    if (this.canInspectTopWindow_()) {
```

Comment:
Let's gate this new behavior on an experiment so we can test it.

Reply:
Done. Will create an issue for cleaning up later.